### ISSUE TITLE
Custom App Lyrics shows "Something went wrong" error

### ISSUE THREAD

**Issue by Utsav-Murudi** [spicetify/spicetify-cli#1491]
### 🔍 Have you checked Spicetify.app page for your issue?

- [X] I have checked the FAQ

### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### ℹ Environment / Computer Info

```Markdown
- Spotify version: Latest version
- Spicetify version: Latest version
```

### 📝 Description

Tried restarting and reinstalling multiple times but still getting the "Something went wrong, Try reloading the page" error

### 📸 Screenshots

![image](https://user-images.githubusercontent.com/76831372/154857883-1bd4e0f9-c6fc-45b7-9406-5d3ec77f65c8.png)

**Comment by CharlieS1103** [comment]
In lines that have document.querySelector("main .os-viewport") just replace it with document.querySelector(".os-viewport") Checked this on my phone so might be wrong but probs will work

**Comment by Utsav-Murudi** [comment]
In which .js file do I have to do this?

**Comment by guttitto** [comment]
Thanks @CharlieS1103, your fix worked for me on New Releases and Reddit :)

**Comment by afonsojramos** [comment]
This has been fixed in #1510 and #1511, however, we have not made a new release with the fixes yet. Expect it soon.
